quesma/quesma/matchers.go

```diff
@@ -133,6 +133,7 @@ func matchAgainstKibanaInternal() mux.RequestMatcher {
 
 		// 1. https://www.elastic.co/guide/en/security/current/alert-schema.html
 		// 2. migrationVersion
-		return !hasJsonKey("kibana.alert.", q) && !hasJsonKey("migrationVersion", q)
+		// 3., 4., 5. related to Kibana Fleet
+		return !hasJsonKey("kibana.alert.", q) && !hasJsonKey("migrationVersion", q) && !hasJsonKey("idleTimeoutExpiration", q) && !strings.Contains(req.Body, "fleet-message-signing-keys") && !strings.Contains(req.Body, "fleet-uninstall-tokens")
```

Comment:
These keys could be extracted as consts

Reply:
Those values are only used here once, I don't think we can give them good variable names (they are a bit of hack, we came up with them by looking at Kibana's traffic, trial-and-error approach).

So for now I'll leave it as-is, hardcoded here.